### DISCUSSION TITLE
Hot reload MST

### DISCUSSION THREAD

**Discussion by midrizi** [infinitered/ignite#2000] [Q&A]
As of right now changing anything in MST stores will not work with hot reload, sometimes it stops updating the UI and sometimes doesn't reflect those changes without fully reloading the app. Wondering if there is a way to get that working?

Maybe root store could be re-initialized on hot reload?

**Comment by jamonholmgren** [comment]
Good question! I’ll think about this and get back with you. Also, congrats on issue/discussion number 2000! 😅

Here’s the docs for anyone interested in pitching in ideas: https://reactnative.dev/docs/fast-refresh
